### DISCUSSION TITLE
Remove buff - Stop Buff Apply doesen't work

### DISCUSSION THREAD

**Discussion by AceOfSettlers** [fedorovvl/tso_client#58] [Q&A] [answered]
Hello @fedorovvl , Thanx for the add of Remove buff recording 

it record the buff , but when you want to apply it it says Already buffed so it doesen't delet the buff 
![image](https://user-images.githubusercontent.com/114313240/196387255-83bbafdf-543d-4f49-9be3-c17c2afd865c.png)

**Comment by igorSof** [comment]
да, удалитель усилителя (RemoveBuff) не накладывается, так как здание под бафом уже находится.

**Reply by AceOfSettlers** [reply]
@igorSof the purpose of the remove buff is to remove this buff

**Comment by AceOfSettlers** [comment, marked as answer]
Fixed by @fedorovvl in the last version of the Client , Now it's working
